kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/registry/Registry.kt

```diff
@@ -56,18 +58,16 @@ class Registry(val jvmModuleName: String = defaultJvmModuleName) {
     ClassGraph()
       .enableAllInfo() // Scan classes, methods, fields, annotations
       .acceptPackages(jvmModuleName)
-      .scan().use { scanResult ->
-        // Use the ScanResult within the try block, e.g.
-        for (clazz in scanResult.getClassesWithMethodAnnotation(Verb::class.java)) {
-          val kClass = clazz.loadClass().kotlin
-          if (kClass.hasAnnotation<Ignore>()) return
-          clazz.methodInfo
-            .filter { info -> info.hasAnnotation(Verb::class.java) && !info.hasAnnotation(Ignore::class.java) }
-            .forEach { info ->
-              val function = info.loadClassAndGetMethod().kotlinFunction!!
-              maybeRegisterVerb(kClass, function)
-            }
+      .scan()
+      .getClassesWithMethodAnnotation(Verb::class.java)
+      .forEach {
+        val kClass = it.loadClass().kotlin
+        if (kClass.hasAnnotation<Ignore>()) {
+          return@forEach
```

Comment:
What's this magical syntax?!?

Reply:
usually it's not necessary but in the odd case comes in handy!